homeassistant/components/sensor/nest.py

```diff
@@ -43,22 +52,45 @@
 
 SENSOR_TEMP_TYPES = ['temperature', 'target']
 
+_SENSOR_TYPES_DEPRECATED = SENSOR_TYPES_DEPRECATED + \
+        list(DEPRECATED_WEATHER_VARS.keys()) + PROTECT_VARS_DEPRECATED
+
 _VALID_SENSOR_TYPES = SENSOR_TYPES + SENSOR_TEMP_TYPES + PROTECT_VARS + \
                       list(WEATHER_VARS.keys())
 
+_VALID_SENSOR_TYPES_WITH_DEPRECATED = _VALID_SENSOR_TYPES + \
+        _SENSOR_TYPES_DEPRECATED
+
 PLATFORM_SCHEMA = vol.Schema({
     vol.Required(CONF_PLATFORM): DOMAIN,
     vol.Optional(CONF_SCAN_INTERVAL):
         vol.All(vol.Coerce(int), vol.Range(min=1)),
-    vol.Required(CONF_MONITORED_CONDITIONS): [vol.In(_VALID_SENSOR_TYPES)],
+    vol.Required(CONF_MONITORED_CONDITIONS): [vol.In(_VALID_SENSOR_TYPES_WITH_DEPRECATED)],
 })
 
+_LOGGER = logging.getLogger(__name__)
+
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup the Nest Sensor."""
     nest = hass.data[DATA_NEST]
     conf = config.get(CONF_MONITORED_CONDITIONS, _VALID_SENSOR_TYPES)
 
+    for variable in conf:
```

Comment:
I'm not sure what hound is trying to tell me here. It seems like a single statement to me, unless my python is bad enough it's really confusing hound.